.gitignore

```diff
@@ -36,6 +36,3 @@ dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
 # https://github.com/takari/maven-wrapper#usage-without-binary-jar
-.mvn/wrapper/maven-wrapper.jar
```

Comment:
Now you've deleted some ignores :D

Reply:
See https://github.com/kit-sdq/programming-lecture-eclipse-artemis/pull/138/files

Reply:
Sorry, you have to fix the last line, then I'll merge the PR 😄

Reply:
The current version is the same as in main? I think?

Reply:
Newline is missing as far as I see :D